includes/amp-helper-functions.php

```diff
@@ -1608,6 +1608,12 @@ function amp_get_content_sanitizers( $post = null ) {
 		);
 	}
 
+	$is_auto_lightbox_disabled = apply_filters( 'amp_auto_lightbox_disabled', true );
+
+	if ( ! $is_auto_lightbox_disabled ) {
```

Comment:
This is an opposite of what should be. If `$is_auto_lightbox_disabled` is true, we should use `AMP_Auto_Lightbox_Disable_Sanitizer`.

Reply:
Yes, I think it should be vice versa.
However current implementation is based on the Implementation brief of the issue.

Reply:
What I meant here is that the variable name says `is_auto_lightbox_disabled ` and when it was `true` (i.e. `disabled`) we actually _enabled_ the auto lightbox feature - so the opposite.

Reply:
What I meant here is that the variable name says `is_auto_lightbox_disabled ` and when it was `true` (i.e. `disabled`) we actually _enabled_ the auto lightbox feature - so the opposite.